presto-docs/src/main/sphinx/release/release-0.287.rst

```diff
@@ -0,0 +1,98 @@
+
+Security Changes
+________________
+* Remove logback 1.2.3. :pr:`21819`
+* Add session property ``default-view-security-mode`` to choose the default security mode for view creation. :pr:`21956`
+
+Verifier Changes
+________________
+* Add support for extended bucket verification of INSERT and CTAS queries. This can be enabled by the configuration property ``extended-verification`` to verify each bucket's data checksum if the inserted table is bucketed. :pr:`22001`
+* Add support for extended partition verification of INSERT and CTAS queries. This can be enabled by the configuration property ``extended-verification`` to verify each partition's data checksum if the inserted table is partitioned. :pr:`21983`
+
+SPI Changes
+___________
+* Add replaceColumn method to com.facebook.common.Page. :pr:`22493`
+* Remove SPI method ConnectorMetadata.getTableLayouts() as deprecated. Add ConnectorMetadata.getTableLayoutForConstraint() as replacement. :pr:`21933`
+* Move `SortNode` to SPI module to be utilized in connector. :pr:`22497`
+
+Hive Changes
+____________
+* Fix a potential wrong results bug when footer stats are marked unreliable and partial aggregation pushdown is enabled.  Such queries will now fail with an error. :pr:`22011`
+* Improve the ``hive.orc.use-column-names`` configuration setting to no longer fail on reading ORC files without column names, but fall back to using Hive's schema. This change improves compatibility with legacy ORC files. :pr:`21391`
+* Add session property ``hive.dynamic_split_sizes_enabled`` to use dynamic split sizes based on data selected by query.  :pr:`22051`
+* Add support for Filelist caching for symlink tables.  :pr:`19145`
+* Add $row_id as a new hidden column. :pr:`22008`
+* Add system procedure ``system.invalidate_directory_list_cache()`` to invalidate directory list cache in Hive Catalog. :pr:`19821`
+
+Iceberg Changes
+_______________
+* Fix identity and truncate transforms on DecimalType columns. :pr:`21958`
+* Fix the bug that ``CAST`` from non-legacy timestamp to date rounding to future when the timestamp is prior than `1970-01-01 00:00:00.000`. :pr:`21959`
+* Add support to set ``commit.retry.num-retries`` table property with table creation to make the number of attempts to make in case of concurrent upserts configurable. :pr:`21250`
+* Add year/month/day/hour transforms both on legacy and non-legacy TimestampType column. :pr:`21959`
+* Fix error encountered when attempting to execute an ``INSERT INTO`` statement where column names contain white spaces. :pr:`21827`
+* Add support for row-level deletes on Iceberg V2 tables. The delete mode can be changed from ``merge-on-read`` to ``copy-on-write`` by setting table property ``delete_mode``. :pr:`21571`
+* Add support for Iceberg V1 tables in Prestissimo. :pr:`22013`
+* Add support to read Iceberg V2 tables with Position Deletes in Prestissimo. :pr:`21980`
+* Add support for Iceberg concurrent insertions. :pr:`21250`
+
+Mysql Changes
```

Comment:
```suggestion
MySQL Connector Changes
```
Following the [Section naming](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#section-naming) guidelines in Release Notes Guidelines.